docs/security-model.mdx

```diff
@@ -0,0 +1,55 @@
+sidebar_label: Ory Security Model
+---
+
+Ory has a proven security model which can take a moment to get used to. Compared to other solutions, Ory manages sessions, tokens,
+and cookies for you whenever possible. You do not need to worry about refreshing tokens or figuring out whether to store them in
+`localStorage` or `document.cookies`. Instead, Ory solves all of this for you and deploys best-in-class protections for common
+browser attack vectors (XSS, CSRF, ...).
+
+## Ory's Identity Service Need Access to Your Domain
+
+Ory bears the responsibility of managing sessions, cookies, and tokens for you. To do this, Ory needs to be exposed on your
+domain, otherwise HTTP Cookies will not be sent. Per default, Ory's APIs are available at
+
+```
+https://<project-slug>.projects.oryapis.com
+```
+
+Let's say you are developing an app and it runs on `http://localhost:3000`. For Ory to be able to manage the session information,
+it needs to be able to set an HTTP Cookie for `domain=localhost` as well! The same applies when your app runs in production (for
+example on `https://app.example.org`). Here too Ory needs to be able to set the HTTP Cookie for `domain=example.org`:
+
+1. During **development** you use the [Ory Tunnel on your machine](guides/local-development.mdx) to "mirror" the APIs available at
+   `https://<project-slug>.projects.oryapis.com` on your `localhost` domain. Ory offers SDKs for certain deployment options such
+   as [Vercel](guides/protect-page-login/nextjs.mdx) which mirror Ory's APIs without the need of running another process.
+2. For **production** you set up a [Custom Domain](guides/custom-domains.mdx) (for example `https://ory.example.org`) which makes
+   Ory's APIs available at `https://ory.example.org`.
+
+If you're interested in learning more about this topic, head over to
+[HTTP Cookies Explained](further-reading/http-cookies-explained.mdx)
+
+## Frequently Asked Questions
+
+Below we have collected some frequently asked questions about Ory's security model.
+
+### How do I solve CSRF Issues?
+
+### What about OAuth 2.0 / OpenID Connect?
+
+Ory has an OAuth 2.0 and OpenID Connect product currently available as self-hosted open source under the name of
+[Ory Hydra](https://github.com/ory/hydra). Ory thus supports OAuth 2.0 and OpenID Connect, but only recommends it for targeted use
+cases. If you haven't, read
+[Why you probably do not need OAuth2 / OpenID Connect](https://www.ory.sh/oauth2-openid-connect-do-you-need-use-cases-examples/).
+
+### What about Access Tokens / Refresh Tokens?
+
+Ory has an OAuth 2.0 / OpenID Connect product which can generate Access and Refresh Tokens. Unless
+[you need OAuth 2.0 / OpenID Connect](https://www.ory.sh/oauth2-openid-connect-do-you-need-use-cases-examples/), Ory's Security
+Model is a much easier solution to handle.
+
+### What about JSON Web Tokens?
+
+There is an [open issue to support JSON Web Tokens](https://github.com/ory/kratos/issues/2487) for Ory's Identity Service.
```

Comment:
I'd rather say that "currently, Ory doesn't support JWTs". When this is actually implemented we can easily update the docs.  

This way we avoid giving any hope / promising a feature (see the "no-promises policy" :) ).

Reply:
Ok, we do want our users input on this though. So how about "let us know about your use case" and link to the issue?